elixir/apps/domain/test/support/new_fixtures/subject_fixtures.ex

```diff
@@ -59,11 +59,17 @@ defmodule Domain.SubjectFixtures do
             actor: actor
           })
 
-        token_fixture(token_attrs)
+        token = token_fixture(token_attrs)
+        token
```

Comment:
Just curious, did the return from the `token_fixture()` not work in the anonymous function for some reason?

Reply:
Ah this is leftover from the refactor to return `{token, encoded}` which was reverted. Fixed.